src/plugins/workspace/common/constants.ts

```diff
@@ -4,6 +4,8 @@
  */
 
 import { i18n } from '@osd/i18n';
+import { DATA_SOURCE_SAVED_OBJECT_TYPE } from '../../data_source/common/data_sources';
```

Comment:
i see that `data_source/common/index` exports this type can't we export the connection type there as well and just import from the like  
```
import { DATA_CONNECTION_SAVED_OBJECT_TYPE, DATA_SOURCE_SAVED_OBJECT_TYPE } from '../../data_source/common'
```

Reply:
Yes, i had tried this way. But it will throw error
`Module not found: Error: import [../../data_source/common] references a non-public export of the [dataSource] bundle and must point to one of the public directories: [public, common/data_sources, common/util, common/data_connections]` because of this config(https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data_source/opensearch_dashboards.json#L9).

Reply:
Updated.